src/test/java/org/exmple/utility/UtilityMethodsTest.java

```diff
@@ -3,10 +3,7 @@
 import org.example.utility.UtilityMethods;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.*;
```

Comment:
We should not use wildcard imports. As an example see "[Google Java Style Guide: 3.3.1 No wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)". We should use explicit imports instead.

For your interest see [Why is using a wild card with a Java import statement is bad](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad)